apps/style/common.scss

```diff
@@ -414,8 +414,8 @@ html[dir='rtl'] div#visualizationResizeBar {
 }
 
 .karel-counter-text.paint {
```

Comment:
is this class only used for paint buckets? I think it is but want to double check

Reply:
Pretty sure, class gets used here, which comment says is only used for paint buckets:

https://github.com/code-dot-org/maze/blob/063e78acb02061d20541c16058160577bd324ea3/src/neighborhoodDrawer.js#L497